lightningd/options.c

```diff
@@ -568,10 +571,17 @@ static void check_config(struct lightningd *ld)
 
 static void setup_default_config(struct lightningd *ld)
 {
-	if (get_chainparams(ld)->testnet)
-		ld->config = testnet_config;
-	else
-		ld->config = mainnet_config;
+	if (strstr(get_chainparams(ld)->network_name, "groestlcoin")) {
```

Comment:
Since we are plain c-grs-lightning, we should move back using to mainnet_config and testnet_config to make it easier to merge with upstream